<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_search_delivery</class>
 <widget class="QWidget" name="admin_search_delivery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1229</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查询快递信息</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
                QLabel {
                    color: #333;
                    font: 14px &quot;幼圆&quot;;
                    font-weight: bold;
                }

                QLineEdit,
                QComboBox  {
                    padding: 8px;
                    font-size: 14px;
                    border: 1px solid #ccc;
                    border-radius: 4px;
                }

                QTableWidget {
                    font: 14px &quot;幼圆&quot;;
                    background-color: #cdefff;
                }

                QTableWidget::item {
                    font: 14px &quot;幼圆&quot;;
                }

                QHeaderView {
                    font: bold 16px &quot;幼圆&quot;;
                }
                
                QTableWidget::item:selected {
                    background-color: #a6a6a6;
                }

                QTableWidget QHeaderView::section {
                    background-color: #65a3ff;
                    color: white;
                    font-weight: bold;
                    font-size: 16px;
                    padding: 4px;
                }

                QTableWidget::item {
                    padding: 10px;
                }

                QPushButton {
                    padding: 8px 16px;
                    font-size: 15px;
                    font-weight: bold;
                    border: 1px solid #4CAF50;
                    border-radius: 4px;
                    color: #fff;
                    background-color: #4CAF50;
                }

                 QPushButton:hover {
                    background-color: #45a049;
                    border: 1px solid #45a049;
                }

                QTextEdit {
                    font-size: 14px;
                    border: 1px solid #ccc;
                    border-radius: 4px;
                }

                QScrollBar:vertical {
                    border: 1px solid #ccc;
                    background: #f0f0f0;
                    width: 10px;
                    margin: 0px 0px 0px 0px;
                }

                QScrollBar::handle:vertical {
                    background: #4CAF50;
                    min-height: 20px;
                }
            </string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1201</width>
     <height>661</height>
    </rect>
   </property>
   <property name="title">
    <string>快递信息</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>1181</width>
      <height>441</height>
     </rect>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>快递单号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>配送员id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>快递员id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>寄出省</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>寄出市</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>寄出详细地址</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>寄件人姓名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>寄件人电话</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>收件省</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>收件市</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>收件详细地址</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>收件人姓名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>收件人电话</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>备注</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>当前位置</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>当前状态</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>取件码</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>预计送达时间</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>完整路线</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>510</y>
      <width>621</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>快递单号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="parcelIdInput"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>收件人电话：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="telInput"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>560</y>
      <width>621</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>配送员ID：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="deliveryIdInput"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>快递员ID：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="postIdInput"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>610</y>
      <width>951</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>寄出省:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sendPrvBox">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>寄出市:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sendCityBox">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>收件省:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="rePrvBox">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>收件市:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="reCityBox">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchBtn">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="returnBtn">
    <property name="geometry">
     <rect>
      <x>1057</x>
      <y>10</y>
      <width>131</width>
      <height>37</height>
     </rect>
    </property>
    <property name="text">
     <string>返回上一级</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
